README.md

```diff
@@ -10,6 +10,9 @@ comments.  For more information about OpenShift, visit https://openshift.redhat.
 or the OpenShift support page
 https://openshift.redhat.com/support.
 
+RHC is typically built on Ruby 1.8.7.  If you are running into any dependency 
+issues when using RHC with later versions of Ruby, please try building the RHC 
```

Comment:
What we want to emphasise is that the Gemfile has some conditionals that modify the dependencies on the net-ssh and addressable gems at buildtime, and RHC built with one version of Ruby may not necessarily work on another version of Ruby.  Maybe change this to "when using one version of Ruby to run an RHC gem that was built using a different version of Ruby".